cfgmgr/buffer_check_headroom_mellanox.lua

```diff
@@ -22,6 +22,8 @@ end
 -- Initialize the accumulative size with 4096
 -- This is to absorb the possible deviation
 local accumulative_size = 4096
+-- Egress mirror size: 2 * maximum MTU (10k)
+local egress_mirror_size = 20*1024
```

Comment:
Is it 10K or 20K? comments don't match the calculation

Reply:
For each port, it is 20k.
The comments mean 2 * 10=20